curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-069.md

```diff
@@ -8,7 +8,10 @@ isHidden: true
 ## Description
 <section id='description'>
 
-Using the same style selector in the previous step, make the font size of the items and prices by using a value of `18px`.
+The default properties of an `hr` element will make it appear as a thin light grey line. You can change the heighth of the line by specifying a value for the `height` property.
```

Comment:
The default properties of an `hr` element will make it appear as a thin light grey line. You can change the height of the line by specifying a value for the `height` property.